plugins/ua_config_standard.c

```diff
@@ -9,6 +9,7 @@
 #include "ua_securitypolicy_none.h"
 #include "ua_types.h"
 #include "ua_types_generated_handling.h"
+#include "ua_client_highlevel.h"
```

Comment:
@jpfr Should we put the `UA_SubscriptionSettings` struct into another more fitting header? E.g. `ua_server_config.h` to avoid including client functionality here.